web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/CitizenCertificateRecipientConverterImpl.java

```diff
@@ -0,0 +1,43 @@
+package se.inera.intyg.intygstjanst.web.service.impl;
+
+import org.springframework.stereotype.Service;
+import se.inera.intyg.intygstjanst.web.service.CitizenCertificateRecipientConverter;
+import se.inera.intyg.intygstjanst.web.service.bean.CertificateRecipientType;
+import se.inera.intyg.intygstjanst.web.service.dto.citizen.CitizenCertificateRecipientDTO;
+import se.inera.intyg.intygstjanst.web.service.repo.RecipientRepo;
+
+import java.time.LocalDateTime;
+import java.util.stream.Collectors;
+
+@Service
+public class CitizenCertificateRecipientConverterImpl implements CitizenCertificateRecipientConverter {
+
+    private final RecipientRepo recipientRepo;
+
+    public CitizenCertificateRecipientConverterImpl(RecipientRepo recipientRepo) {
+        this.recipientRepo = recipientRepo;
+    }
+
+    @Override
+    public CitizenCertificateRecipientDTO get(String certificateType, LocalDateTime sent) {
```

Comment:
Instead of using null, when we know it might not return a value, could you use Optional? This is a comment that can be considered in other places as well.

Reply:
Fixed for relation and recipient converter